common/include/km_types.h

```diff
@@ -1,5 +1,7 @@
 #pragma once
 
+#include <stdint.h>
```

Comment:
Curious why this is needed?

Reply:
@mcdurdin me too

Reply:
i wonder if it used to be indirectly included somehow.  That's the danger of optimizing includes !

Reply:
So you found you needed it to build?

Reply:
Jenkins build is failing because of this line...

```
[2022-08-24T18:58:13.334Z] /<<PKGBUILDDIR>>/common/include/km_types.h:3:10: fatal error: cstdint: No such file or directory

[2022-08-24T18:58:13.334Z]  #include <cstdint>
```

Reply:
for linux, yes.  I don't see a teamcity build that shows this, because it stopped on an earlier error. I can test that though

Reply:
^ @mcdurdin  that was old, i changed <cstdint> to <stdint.h> 

I opened https://github.com/keymanapp/keyman/pull/7117 to try skipping that line, to double check that it would fail on linux not just on srl-linux

Reply:
i'll merge #7116 *unless* #7117  shows that the #include is unnecessary

Reply:
^ windows failed without that include… passed with.